api/dns/settings.py

```diff
@@ -124,8 +124,7 @@
 # https://docs.djangoproject.com/en/5.1/howto/static-files/
 
 STATIC_URL = 'static/'
-static_root_path = os.getenv("SNAP_DATA") or BASE_DIR
-STATIC_ROOT = os.path.join(static_root_path, '/api/static/')
+STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
```

Comment:
Do you think it would be better to use Pathlib: https://docs.python.org/3/library/pathlib.html?